good_first_issues/utils/__init__.py

```diff
@@ -104,6 +104,17 @@ def identify_limit(limit: Optional[int], all: bool) -> Optional[int]:
         return 10
 
 
+def get_row_ids(issue_count, limiter):
+    """
+    Generate custom Row Index Column for the table
+    Returns a Iterable.
+    """
+    rowIDs = []
```

Comment:
change to `row_ids` to match snake case variable naming convention of code base.

Reply:
done